app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
if we don't override anything in confirmation controller why do we need this file
in addition if we want use it we should change routes likes you did for registration controller